localdata.js

```diff
@@ -0,0 +1,17 @@
+const contactForm = document.forms[0];
+const formValues = {};
+if (window.localStorage.getItem('show data')) {
+  const formObject = JSON.parse(window.localStorage.getItem('show data'));
+  contactForm.name.value = formObject.name;
+  contactForm.lname.value = formObject.lname;
+  contactForm.email.value = formObject.email;
+  contactForm.message.value = formObject.message;
+}
+function collectFormData() {
+  formValues.name = contactForm.name.value;
+  formValues.lname = contactForm.lname.value;
+  formValues.email = contactForm.email.value;
+  formValues.message = contactForm.message.value;
+  window.localStorage.setItem('show data', JSON.stringify(formValues));
+}
+contactForm.addEventListener('submit', collectFormData);
```

Comment:
- [ ] Good job so far 👍 However, the requirements state that the input data is saved upon change and not upon submitting the form. To explain it more, when the user changes the content of any input field in the form the data object is updated and it is saved to the local storage. So, when you put values in the input fields then you refresh (without submitting the form) the values will still be there in the input fields 💯 Kindly make the necessary changes to meet the project requirements 😄